libs/ledger/src/chain/block_coordinator.cpp

```diff
@@ -1106,20 +1112,25 @@ BlockCoordinator::State BlockCoordinator::OnReset()
 
   if (next_block_)
   {
+    std::cerr << "wasn " << next_block_->body.block_number << std::endl;  // DELETEME_NH
```

Comment:
Remove?

Reply:
Uhhh yes. I would say good spot but it was like a sore thumb